test/data/testAppObj.json

```diff
@@ -525,6 +525,7 @@
           "job_state": "queued",
           "result": [],
           "status": "queued",
+          "created": 1607109147000,
```

Comment:
added `created` timestamps to make sure that they are considered valid job state objects